neural_compressor/experimental/compression/__init__.py

```diff
@@ -15,4 +15,48 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from .pruning import prepare_pruning
+
+from os.path import dirname, basename, isfile, join
+import glob
+from .pruning import PRUNINGS
+from .utils import torch
+from typing import Optional
+# modules = glob.glob(join(dirname(__file__), "*.py"))
+
+# for f in modules:
+#     if isfile(f) and not f.startswith('__') and not f.endswith('__init__.py'):
+#         __import__(basename(f)[:-3], globals(), locals(), level=1)
+
+FRAMEWORK = {
+    'pytorch': 'pt',
+    'keras': 'keras'
+}
+
+def get_pruning(config, model: torch.nn.Module, opt: Optional[torch.optim.Optimizer] = None,
```

Comment:
do not expose new API to users, we need to review API design with Feng and other teams once we change it.  So better follow the existing API, like prepare_pruning or perpare_compression, and hide the new features.